Garllon/graphql_connector
# Httparty adapter options

Allows to set an optional `httparty_adapter_options` like `timeout` when interacting with a graphql API

### Manual testing
* Start a graphql server
* Use this MR of `graphql_connector` 
* Perform a graphql-query against the server and expect to receive some response based on your query
* Start `graphql_connector` again but this time with `httparty_adapter_options` set (e.g. `timeout`)
* Expect any graphql-query against the server to consider `httparty_adapter_options`

### Why
We realized that especially server timeouts could case the `connector` be in the state of hanging and waiting. Usually it is best to actually **fail** after some timeout is reached (this allows also to think of some backup strategy like a retry logic, default value, ...).
